components/gitpod-db/src/typeorm/workspace-db-impl.ts

```diff
@@ -224,6 +224,9 @@ export abstract class AbstractTypeORMWorkspaceDBImpl implements WorkspaceDB {
         if (options.pinnedOnly) {
             qb.andWhere("ws.pinned = true");
         }
+        if (options.organizationId) {
```

Comment:
Why do we still need the non-org filtered version of this API?

Reply:
I guess the DB layer is fine with this being optional, but would've expected that `orgID` would be made required on the `ws API layer`

Reply:
We haven't migrated everyone so the organizationId is not set on all workspaces. Therefore I plan to roll this out only to migrated users. Once we have migrated everyone we should go back to the code and make this mandatory.